app/jobs/appointment_reminders/send_reminder_job.rb

```diff
@@ -0,0 +1,103 @@
+class AppointmentReminders::SendReminderJob
+  include Rails.application.routes.url_helpers
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :high
+
+  def perform(reminder_id)
```

Comment:
From scanning this job it duplicates lots of logic from the existing notification worker.  Another reason to migrate the existing code first.